package.json

```diff
@@ -88,6 +88,11 @@
           "type": "boolean",
           "default": true,
           "description": "By default, when invoking `editor.action.revealDefinition` on `[[note.md]]` if `note.md` does not exist in workspace, create it. NB: Works only when `vscodeMarkdownNotes.workspaceFilenameConvention` = 'uniqueFilenames'."
+        },
+        "vscodeMarkdownNotes.newNoteTemplate": {
+          "type": "string",
+          "default": "# ${noteName}\n\n",
+          "description": "template for auto-created note files. Available tokens: ${noteName}, ${timestamp}"
```

Comment:
(1) nit: capitalize the "t" in template in the first word of the description
(2) add an example of the timestamp at the end of the description maybe? " (2020-07-09T05:29:00.541Z)."

Reply:
Done.